config/locales/common.yml

```diff
@@ -19,8 +19,11 @@ en:
       edit: Edit
       manage: Manage
       next: Next
+      no_label: 'No'
```

Comment:
`i18n-task normalize` changes `no:` to `false` and `yes` to `true` for some reason 🤷‍♂️  hence the `_label` append here